lib/src/box.dart

```diff
@@ -9,8 +9,9 @@ import 'bindings/helpers.dart';
 import 'bindings/structs.dart';
 import 'modelinfo/index.dart';
 import 'query/query.dart';
+import 'relations/to_one.dart';
 
-enum _PutMode {
+enum PutMode {
```

Comment:
So why is this now public again?

Reply:
hmm, I originally used it in to_one.dart (put was forwarded through there, instead of returning `internalTargetBox`). 
I could roll-back the change as it is unrelated to this PR, but I'd actually suggest keeping it anyway. We've already accepted mode param in `box.put()` but actually didn't allow users to pass any value :facepalm:

Reply:
Ah, ok. So users should be able to set this, then all good!